src/main/java/Duke.java

```diff
@@ -1,10 +1,238 @@
+    }
+
+    public static void displayList(){
+        System.out.println(boundary);
+
+        for (int i = 0; i < itemList.size(); i++) {
+
+            System.out.println(String.format("%d", index) + "." + itemList.get(i));
+            index++;
+        }
+        System.out.println(boundary);
+        index = 1;
+    }
+    public static void done(){
+        int doneTaskNo = Integer.parseInt(Message[1]);
+        task = itemList.get(doneTaskNo - 1);
+        task.markAsDone();
+        System.out.println("Nice! I've marked this task as done: ");
+        System.out.println("\t[" + task);
+        System.out.println(boundary);
+    }
+    public static void deleteTask() {
+        int deleteTaskNo = Integer.parseInt(Message[1]);
+        task = itemList.get(deleteTaskNo - 1);
+        itemList.remove(task);
+        System.out.println("Noted. I've removed this task:");
+        System.out.println("  " + task);
+        System.out.println("Now you have " + (taskNo - 1) + " task in the list\n" + boundary);
+        taskNo--;
+    }
+    public static void addTodo(){
+        try {
+            newTask = new ToDo(Message[1]);
+            itemList.add(newTask);
+            System.out.println(boundary + "\nGot it. I've added this task: " + System.lineSeparator()
+                    + "  " + itemList.get(taskNo));
+            System.out.println("Now you have " + (taskNo + 1) + " task in the list\n" + boundary);
+            taskNo++;
+        } catch (IndexOutOfBoundsException e) {
+            System.out.println("Oops!!! The description of todo cannot be empty");
+        }
+    }
+    public static void addDeadline(){
+        try {
+            String[] DeadlineMessage = Message[1].split("/by ", 2);
+            newTask = new Deadline(DeadlineMessage[0], DeadlineMessage[1]);
+            itemList.add(newTask);
+            System.out.println(boundary + "\nGot it. I've added this task: " + System.lineSeparator()
+                    + "  " + itemList.get(taskNo));
+            System.out.println("Now you have " + (taskNo + 1) + " task in the list\n" + boundary);
+            taskNo++;
+        }catch(IndexOutOfBoundsException e){
+            System.out.println("Oops!!! Please enter a description and deadline");
+        }
+    }
+
+    public static void addEvent(){
+        try {
+            String[] EventMessage = Message[1].split("/at", 2);
+            newTask = new Events(EventMessage[0], EventMessage[1]);
+            itemList.add(newTask);
+            System.out.println(boundary + "\nGot it. I've added this task: " + System.lineSeparator()
+                    + "  " + itemList.get(taskNo));
+            System.out.println("Now you have " + (taskNo + 1) + " task in the list\n" + boundary);
+            taskNo++;
+        }catch(IndexOutOfBoundsException e){
+            System.out.println("Oops!!! Please enter a description and event date");
+        }
+    }
+    public static void Help(){
```

Comment:
You might want to follow Java coding standards' convention of naming methods in camelCase i.e. help(), or maybe showHelp()? Just like how you define done() in line 173.